boards/arm/twr_ke18f/pinmux.c

```diff
@@ -34,7 +34,7 @@ static int twr_ke18f_pinmux_init(struct device *dev)
 		device_get_binding(CONFIG_PINMUX_MCUX_PORTE_NAME);
 #endif
 
-#if DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(ftm0), nxp_kinetis_ftm_pwm, okay)
+#if DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(ftm0), nxp_kinetis_ftm_pwm, okay) && CONFIG_PWM
```

Comment:
For the twr_ke18f, you’re missing a guard for the user LEDs further down pinmux.c.

Reply:
fixed